zipkin-api.yaml

```diff
@@ -0,0 +1,259 @@
+swagger: "2.0"
+info:
+  version: "1.0.0"
+  title: Zipkin API
+  description: |
+    Zipkin's Query api is rooted at `api/v1`, on a host that by default listens
+    on port 9411. It primarily serves zipkin-web, although it includes a POST
+    endpoint that can receive spans.
+host: localhost:9411
+basePath: /api/v1
+schemes:
+  - http
+  - https
+consumes:
```

Comment:
That one is in here https://github.com/mohsen1/zipkin-api-1/commit/35a5531dc635a3aa9f4f943b6f080d23a053b08f#diff-bee5d8ba70d2a6b769696a3afe4341ccR141

The rest is JSON in, JSON out right?